test/e2e/spec/e2e_spec.rb

```diff
@@ -129,6 +129,19 @@
 
     end
 
+    describe "Shelley Migrations" do
+      it "I can create migration plan shelley -> shelley" do
+        target_id = create_shelley_wallet
+        addrs = SHELLEY.addresses.list(target_id).map { |a| a['id'] }
+
+        plan = SHELLEY.migrations.plan(@wid, addrs)
```

Comment:
Yes, it has some non-ADA assets. It is fixture Shelley wallet used in e2e testnet tests.

Reply:
In that case, I wonder if it would be possible to add an assertion about `balanceSelected`, to assert that the set of unique assets is _not_ equal to `[]`?

Reply:
done in 091e063.